SKIP-REQUIRE-JAVADOC

```diff
@@ -0,0 +1 @@
+Whoever merges this pull request should remove this file when the pull request is merged.
```

Comment:
I'm not auto merging because this needs to be removed after merging, so it must be manually merged.